src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -196,7 +203,12 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
       topStoriesContent ? (
         <ExperimentTopStories topStoriesContent={topStoriesContent} />
       ) : null,
-    jumpTo: JumpTo,
+    jumpTo: (props: ComponentToRenderProps & JumpToProps) =>
+      jumpToVariation === 'on_off' ? (
```

Comment:
Is `on_off` just for initial testing? We could just use the `on` here to test that values are coming through to Optimizely.

In the real world, this will be `on` for showing the component and then we just return `null` for any other variation.